src/renderer/components/image-button/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FC, MouseEventHandler, useContext } from 'react';
+import Image from 'react-bootstrap/Image';
+
+import { LocaleContext } from '../../contexts';
+
+const ImageButton: FC<{ onClick?: MouseEventHandler<HTMLElement>; img: string; passedWidth: string }> = ({
+  onClick,
+  img,
+  passedWidth,
```

Comment:
do you mean when I call the FC on the other pages I should call width there rather than passing it into the image button?

for example: in the back-button index.tsx line 11
<ImageButton onClick={onBack} img={backButton} style={{ width: "1rem"}}></ImageButton>

Reply:
No, I meant rather than naming this prop `passedWidth`, why not name it just `width`?

Reply:
![image](https://user-images.githubusercontent.com/71040820/101531074-c1713e80-3960-11eb-8ac1-e8123133f157.png)
not sure if that image attached to the comment correctly
but when I name the prop width I get the following error and the only way I've found to fix it is to name the variable something else

  ✘  http://eslint.org/docs/rules/object-shorthand  Expected property shorthand  
  src\renderer\components\image-button\index.tsx:12:33

Reply:
This is just saying, instead of
```
const something = {width: width};
```


```
const something = {width};
```

Reply:
got this fixed! thanks for the help!